documentation/src/main/asciidoc/stories/assembly_server_monitoring.adoc

```diff
@@ -4,6 +4,7 @@
 
 include::assembly_server_logging.adoc[leveloffset=+1]
 include::assembly_server_health.adoc[leveloffset=+1]
+include::{topics}/proc_scraping_metrics.adoc[leveloffset=+1]
```

Comment:
Please remove 'scraping' references because it hints to the old way of obtaining metrics. 'Exposing' or 'collecting' are appropriate alternatives depending on context.